docs/release-notes.md

```diff
@@ -0,0 +1,23 @@
+# Lockbox Release Notes
+
+## 0.1.0 - alpha
+
+_Date: 2017-10-16_
+
+### What's New
+
+This is a pre-release of the Lockbox password manager for internal Mozilla employees.
+
+This starts as a signed Firefox extension where you can:
+
+* Create a Lockbox account with a master password to lock/unlock your data
+* View entries
+* Search entries
+* Add, edit, and delete entries
+* Copy username and password to clipboard
+* Submit feedback
+
+### Known Issues
+
+* There is no way to reset your Lockbox master password. If you forget your master password, you'll need to create a new profile and start fresh.
```

Comment:
This is a bit misleading; uninstalling and then re-installation resets the master password as well as the local datastore.

Reply:
@m8ttyB thanks.  I must have really screwed up my dev profile -- I was having issues with this, but didn't explore further.